bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableDataSettingsFactory.java

```diff
@@ -222,12 +227,15 @@ private static RetrySettings defaultRetrySettings(BigtableOptions options) {
         .setMaxAttempts(retryOptions.getMaxScanTimeoutRetries());
 
     // configurations for RPC timeouts
+    Duration shortRpcTimeout = ofMillis(options.getCallOptionsConfig().getShortRpcTimeoutMs());
     retryBuilder
-        .setInitialRpcTimeout(ofMillis(options.getCallOptionsConfig().getShortRpcTimeoutMs()))
-        .setMaxRpcTimeout(ofMillis(retryOptions.getReadPartialRowTimeoutMillis()))
+        .setInitialRpcTimeout(shortRpcTimeout)
+        .setMaxRpcTimeout(shortRpcTimeout)
         .setTotalTimeout(ofMillis(options.getCallOptionsConfig().getLongRpcTimeoutMs()));
-
-    // TODO: an option to set RetryOptions#allowRetriesWithoutTimestamp
+    
+    if (retryOptions.allowRetriesWithoutTimestamp()) {
+      LOG.warn("Retries without Timestamp doesn't support, please use Mutation#createUnsafe");
```

Comment:
@ajaaym I remember Solomon told about hbase's `Mutation` but I didn't any unsafe mutation in hbase's `Mutation`.. Could you please tell what method should we log.. or "just not support" is enough?